common/src/main/java/com/wynntils/wynn/model/container/ContainerQueryManager.java

```diff
@@ -143,8 +143,10 @@ public static void onContainerSetContent(ContainerSetContentEvent e) {
             }
         } else {
             // We're done
+            ContainerQueryStep oldStep = currentStep;
```

Comment:
Does `endQuery()` reset `currentStep`? Something in "oldStep" leads my thinking astray and got me confused. Maybe you can call it "lastStep" eller "finalStep"?

Reply:
It does reset, yes.